packages/@react-spectrum/form/docs/Form.mdx

```diff
@@ -79,8 +79,8 @@ technology as a form landmark region to which assistive technology can navigate.
 
 ## Events
 Form accepts an `onSubmit` and `onReset` prop for handling form submission and resets respectively. In cases where your form elements' values are controlled, you must handle
-resetting each element's value to their original default via the callback provided via `onReset`. This is particularly important when using form elements such as RadioGroup and NumberField
-whose values are inherently controlled through their implementation.
+resetting each element's value to their original default via the callback provided via `onReset`. This is particularly important when using form elements such as Radio, RadioGroup,
```

Comment:
I mean, this is going to matter for way more components than it'll just work for. Probably better not to be explicit about any of them and just say it's important.

Other fields I can think of are, ColorField/Area/Wheel, CheckboxGroup, Sliders, Combobox, SearchField, SearchWithin, Picker, not sure if any are problematic or if there are others I'm forgetting

Reply:
Sounds good, updated to remove the list of components